server/datastore/mysql/software.go

```diff
@@ -895,3 +1026,22 @@ ORDER BY
 	}
 	return hosts, nil
 }
+
+func (ds *Datastore) NewCVE(ctx context.Context, cve *fleet.CVE) (*fleet.CVE, error) {
```

Comment:
Why does `Datastore.NewCVE` return the provided `cve`?

Reply:
Consider renaming to `NewCVEScore`.

Reply:
There wasn't a strong reason to return it, I was just following the convention of other `New*` datastore methods.